overlord/snapstate/policy/export_test.go

```diff
@@ -16,7 +16,7 @@ var (
 	ErrSnapdNotRemovableOnCore       = errSnapdNotRemovableOnCore
 	ErrSnapdNotYetRemovableOnClassic = errSnapdNotYetRemovableOnClassic
 
-	ErrEphemeralSnapsNotRemovalable = errEphemeralSnapsNotRemovalable
+	ErrEphemeralSnapsNotRemovalable = errEphemeralSnapsNotRemovable
```

Comment:
```suggestion
	ErrEphemeralSnapsNotRemovable = errEphemeralSnapsNotRemovable
```

Reply:
Thanks, I force-pushed this as the rename was broader due to symbol usage.